src/xrEngine/device.cpp

```diff
@@ -34,6 +34,7 @@ ENGINE_API BOOL g_bRendering = FALSE;
 
 BOOL g_bLoaded = FALSE;
 ref_light precache_light = 0;
+int g_dwFPSlimit = 120;
```

Comment:
In some systems (including mine) high framerates produce EMI that introduces noise on audio outputs, which is quite annoying and you don't want that to happen. That's the first reason.

The second reason is that some users (including me) prefer to have 60 fps instead of 200-500-1000 to make the system consume less power and so operate more quiet.